Userland/Libraries/LibJS/Runtime/CommonPropertyNames.h

```diff
@@ -284,6 +284,7 @@ namespace JS {
     P(indexOf)                               \
     P(indices)                               \
     P(info)                                  \
+    P(intersection)                          \
```

Comment:
after `instant`

Reply:
🔴🐧

Reply:
Ah good point, I wasn't sure how capitals factored into that equation.